core/src/test/java/org/apache/calcite/sql/test/SqlTypeNameTest.java

```diff
@@ -172,7 +173,7 @@ class SqlTypeNameTest {
   @Test void testOther() {
     SqlTypeName tn =
         SqlTypeName.getNameForJdbcType(Types.OTHER);
-    assertEquals(null, tn, "OTHER did not map to null");
+    assertEquals(OTHER, tn, "OTHER did not map to null");
```

Comment:
huh?

Reply:
I remember modifying this line and found the test cases unclear (and I forgot to adapt the error message). The javadoc of getNameForJdbcType states: "corresponding SqlTypeName, or null if the type is not known". As other is kind of a known type, does it makes sense to adapt the test accordingly? I will commit this fix for now.

```java
  @Test void testOther() {
    SqlTypeName tn =
        SqlTypeName.getNameForJdbcType(Types.OTHER);
    assertEquals(OTHER, tn, "OTHER did not map to OTHER");
  }
```